backend/prisma/schema.prisma

```diff
@@ -15,48 +15,50 @@ datasource db {
 
 model Player {
```

Comment:
Does this version of prisma have formatting? Try running `prisma format` in the CLI to see if it changes anything